CMakeLists.txt

```diff
@@ -930,16 +930,32 @@ add_custom_target(lrelease
 
 add_dependencies(mscore pluginDocumentation)
 
-
 ##
-##  For MSVC: set the startup project to be mscore
-##     (requires CMake 3.6.3+ to work, but should be benign on other versions
-##      as it is just setting the value for a property).
+## Miscellaneous Microsoft Visual Studio settings
 ##
 if (MSVC)
+
+   # Force the "install" and "package" targets not to depend on the "all" target.
    set(CMAKE_SKIP_INSTALL_ALL_DEPENDENCY true)
    set(CMAKE_SKIP_PACKAGE_ALL_DEPENDENCY true)
-   set(VS_STARTUP_PROJECT mscore)
+
+   # Set the startup project to "mscore".
+   if (NOT ${CMAKE_VERSION} VERSION_LESS "3.6.0")
```

Comment:
the (now removed) comment above talks about CMake 3.6.3+, so why 3.6.0 here?

Reply:
I went and double-checked, and it turns out the original comment was in error. Support for that property was actually introduced in CMake 3.6.0. (See the [CMake 3.6.0 annoucement](https://cmake.org/pipermail/cmake/2016-July/063889.html)).

Reply:
OK, thanks for checking